modules/ossm-deploying-multiple-control-planes.adoc

```diff
@@ -0,0 +1,158 @@
+// Module included in the following assemblies:
+// install/ossm-using-multiple-control-planes-on-single-cluster.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploying-multiple-control-planes_{context}"]
+= Deploying multiple control planes
+:context: ossm-deploying-multiple-control-planes
+
+This procedure demonstrates how to configure a cluster to host two control planes in separate system namespaces. Each control plane will manage specific application workloads running in designated namespaces. The namespaces are associated with a control plane using revision and discovery selector configurations.
+
+.Prerequisites
+* The {SMProduct} operator is installed.
+* An {istio} CNI resource is created.
+* The `istioctl` binary is installed on your localhost.
+
+.Procedure
+
+. Create a system namespace called `usergroup-1`:
++
+[source,terminal]
+----
+$ oc create namespace usergroup-1
+----
+
+. Add label to the namespace:
++
+[source,terminal]
+----
+$ oc label namespace usergroup-1 usergroup=usergroup-1
+----
+
+. Create a YAML file named `istio-1.yaml`:
++
+.Example configuration
+[source,yaml,subs="attributes,verbatim"]
+----
+kind: Istio
+apiVersion: sailoperator.io/v1alpha1
+metadata:
+  name: usergroup-1
+spec:
+  namespace: usergroup-1
+  values:
+    meshConfig:
+      discoverySelectors:
+        - matchLabels:
+            usergroup: usergroup-1
+  updateStrategy:
+    type: InPlace
+  version: v1.23.0
```

Comment:
The "updateStrategy" and "version" fields here may not be appropriate for all users, and the "version" value will start to look out of date quickly. We should remove these fields as they are not relevant to the example.

I'll double check, but hopefully if they are omitted and the user will just get default values if they did try to copy/paste this.

Reply:
Just checked, these are not required or relevant to the example, so we should remove the lines updateStrategy, type and version.